cmd/oci-runtime-tool/generate.go

```diff
@@ -608,6 +613,18 @@ func parseRlimit(rlimit string) (string, uint64, uint64, error) {
 	return parts[0], uint64(hard), uint64(soft), nil
 }
 
+func parseNamespace(ns string) (string, string, error) {
+	parts := strings.SplitN(ns, ":", 2)
```

Comment:
I've fixed it to check `parts[0]`. The whitelist checking should be done in the generate library, and I'll punt it for now. I mean, we don't actually have tests for `generate` at all -- so maybe we should do that first? :wink: